src/surface.cpp

```diff
@@ -539,9 +552,34 @@ bool SurfacePlane::periodic_translate(const PeriodicSurface* other, Position& r,
 }
 
 BoundingBox
-SurfacePlane::bounding_box() const
+SurfacePlane::bounding_box(bool pos_side) const
 {
-  return {-INFTY, INFTY, -INFTY, INFTY, -INFTY, INFTY};
+  BoundingBox bbox = {-INFTY, INFTY, -INFTY, INFTY, -INFTY, INFTY};
+  if (A_ == 0.0 && B_ == 0.0) {
+    double val = D_ / C_;
+    if (pos_side) {
+      bbox.zmin = val;
+    } else {
+      bbox.zmax = val;
+    }
+  } else if (A_ == 0.0 && C_ == 0.0) {
+    double val = D_ / B_;
+    if (pos_side) {
+    bbox.ymin = val;
+    } else {
+    bbox.ymax = val;
+    }
+  } else if (B_ == 0.0 && C_ == 0.0) {
+    double val = D_ / A_;
+    if (pos_side) {
+    bbox.xmin = val;
+    } else {
+    bbox.xmax = val;
```

Comment:
Agreed, it's fairly unlikely. It might happen if someone is using scripts for a parameterization study or to rotate a model arbitrarily, but those are pretty rare and there are still methods to perform these transformations on axis-aligned planes. I'll remove those.